src/styles.css

```diff
@@ -0,0 +1,1198 @@
+}
+
+.contact-me {
+  background-color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 100px 0 0 0;
+  margin-top: 0%;
+  width: 98%;
+}
+
+.bg-image {
+  background-image: url(../assets/contact-me.png);
+  background-repeat: no-repeat;
+  background-position-x: center;
+  background-size: contain;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+.project-section {
+  background-color: #e5e5e5;
+  width: 90%;
+  padding: 39px;
+}
+
+.contact-title {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  height: 56px;
+  justify-content: center;
+  padding-top: 80px;
+}
+
+.contact-text {
+  padding-left: 20%;
+  padding-right: 20%;
+  margin-bottom: 40px;
+  width: 70%;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #ebebff;
+  justify-content: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 28px;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin: 0 8px;
+  align-items: center;
+}
+
+#username {
+  background: #fff;
+  height: 48px;
+  margin: 0 24px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 0 24px;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 21px;
+  color: #172b4d;
+  width: 70%;
+  margin-bottom: 18px;
+}
+
+#useremail {
+  background: #fff;
+  height: 48px;
+  margin: 0 24px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 0 24px;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 21px;
+  color: #172b4d;
+  width: 70%;
+  margin-bottom: 18px;
+}
+
+#usermessage {
+  height: 170px;
+  left: 24px;
+  margin: 0 24px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 0 24px;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 21px;
+  color: #172b4d;
+  width: 70%;
+}
+
+.alert {
+  color: red;
+  margin-top: 5%;
+  font-family: "poppins", sans-serif;
+}
+
+.contact-form button {
+  height: 48px;
+  width: 148px;
+  margin-left: -15%;
+}
+
+.modal-example {
+  display: none;
+  flex-direction: column;
+  background: #fff;
+  border-radius: 20px;
+  margin: 5%;
+  width: 80%;
+  padding: 10px;
+  position: fixed;
+  animation: smoth-effet-open 0.7s;
+  top: 0;
+  color: #172b4d;
+}
+
+.modal-btn {
+  font-size: 17px;
+  text-align: center;
+  padding: 4px;
+  border-radius: 8px;
+  background: #fff;
+  border: 2px #6070ff solid;
+  outline: none;
+  color: #6070ff;
+  font-weight: 600;
+  margin-right: 5px;
+  margin-top: 0;
+}
+
+.modal-example.show {
+  display: flex;
+}
+
+.modal-example img {
+  width: 65%;
+}
+
+.modal-example .closeModal {
+  font-size: 50px;
+  color: #67798e;
+  border: none;
+  outline: none;
+  background: none;
+}
```

Comment:
- [ ] Great effort working on your modal pop up cards, but currently what you have in your design is quite different from what we have in the Figma design. and i would want you to pay optimum attention to the Figma design to have a wonderful replica of the design in your work. Below is a video recording highligting some of the changes you will have to pay attention to. 👇  

- [ ] Also i notice while working on making your project cards dynamic the design has been disrupted on both the `mobile view` and `desktop view`. I would kindly want you to revisit that, as though it is not part of this milestone but it is very paramount to have it fixed, for you to have a wonderful portfolio project to showcase.

  Video 👇 

https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/78634841/2b4b5bc4-39eb-4fdd-ae7b-1c45df46b09d